src/test_endpoint.py

```
@@ -97,6 +97,69 @@ def test_run_customer_service_missing_api_key():
+    payload = {
+        "subject": "Issue from Python App",
+        "description": "Something went wrong in the external system.",
+        "category": "general_inquiry",  # "billing" | "account_issue" | "general_inquiry" | "refund"
+        "priority": "high",
+        "customer_name": "External System",
+        "customer_email": "system@example.com",
+        "created_by_user_id": SYSTEM_USER_ID,
+    }
+    headers = {"x-api-key": API_KEY, "Content-Type": "application/json"}
+
+    response = requests.post(API_URL, json=payload, headers=headers)
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)